### PR TITLE
[TASK] Add PHP 7.3 support

### PR DESCRIPTION
**Pull request by sabbelasichon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -61,9 +65,17 @@ jobs:
       before_script:
         - composer install
         - composer update
+    - <<: *phpunit
+      php: 7.3
+      env: PHPUnit=latest
+      before_script:
+        - composer install
+        - composer update
```
Comment (simonschaufi) [inline .travis.yml]: I don't understand why `install` and `update` is used here. `composer update` would be enough imho.

Comment (sabbelasichon) [reply .travis.yml]: Yes , this is true. Was just copying it over. Did it. Can you check again?

Comment (t3easy) [reply .travis.yml]: I did this to log in the ci de difference between the locked dependencies and what is new.

Comment (sabbelasichon) [reply .travis.yml]: Too smart. So you think we should still keep it?

Comment (t3easy) [reply .travis.yml]: If composer.lock stays it gives you an overview what would change. If it's dropped it can be just install.

Comment (sabbelasichon) [reply .travis.yml]: @t3easy Good point. For the moment composer.lock stays, but i´ll try to get rid of it soon. So we keep it for the moment.

Comment (sabbelasichon) [reply .travis.yml]: @t3easy @simonschaufi Could you both try to approve the PR?

**Pull request merged**